index.bs

```diff
@@ -408,14 +408,14 @@ Excluding inputs generally include
 However, an event whose only effect is to begin or update a fling or scroll
 gesture is not an excluding input.
 
-The user agent may delay the reporting of layout shifts after a
-<a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event
-until such time as it is known that the event does not begin a fling or scroll
-gesture.
-
-The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> and
-<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a>
-events are also not excluding inputs.
+* When a <a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event or
+a <a href="https://www.w3.org/TR/uievents/#event-type-mousedown">mousedown</a> event is received, the
+user agent starts buffering the layout shifts.
+* When a <a href="https://www.w3.org/TR/pointerevents/#the-pointercancel-event">pointercancel</a> event
+is received, all the accumulated layout shift score in the buffer is reported, and it is not an excluding input.
+* When a <a href="https://www.w3.org/TR/pointerevents/#the-pointerup-event">pointerup</a> event or a
+<a href="https://www.w3.org/TR/uievents/#event-type-mouseup">mouseup</a> event is received, the buffer is reset and
+the layout shift score is not reported. They are excluding inputs.
```

Comment:
is reported right? i.e. remove 'not'.

Also, we may want to clarify 'They are excluding inputs.' It doesn't matter which ones, I think the only one that matters is the latest one. So the mouseup/pointerup event is considered an excluding input and then the shifts are reported (this order matters).

Reply:
ACK. Yes, I was going to say the layout shift entry is report with no score change.